includes/abstracts/abstract-wc-gateway-ppec.php

```diff
@@ -95,7 +95,13 @@ public function enqueue_scripts() {
 		// Image upload.
 		wp_enqueue_media();
 
-		wp_enqueue_script( 'wc-gateway-ppec-settings', wc_gateway_ppec()->plugin_url . 'assets/js/wc-gateway-ppec-settings.js', array( 'jquery' ), wc_gateway_ppec()->version, true );
+		$screen = get_current_screen();
+
+		// Only enqueue the setting scripts on the PayPal Checkout settings screen.
+		if ( true ) {
```

Comment:
Sorry, if this is obvious. I do not understand the reason for this condition!

Reply:
haha oopps. Yeah that was a mistake 🤦‍♂ I've rebased that.